app/src/engineering/java/com/hedvig/app/feature/offer/OfferMockActivity.kt

```diff
@@ -39,49 +40,51 @@ class OfferMockActivity : MockActivity() {
         }
     )
 
+    private val fakeQuoteCartId = QuoteCartId("") // MockOfferViewModel doesn't actually use this
+
     override fun adapter() = genericDevelopmentAdapter {
```

Comment:
I thought we killed this thing 🤔 I think we should just remove it tbh

Reply:
Yeah this is just the state of the release version for now which is pre-merge of the hanalytics stuff which came with such refactors. Which is also the reason why I didn't spend more time other than making the compiler happy. When I merge this back into develop, since this class is gone this change will also be gone. I just didn't want to make such a big diff for a hotfix for a thing that we've fixed already anyway :D